pkgs/servers/gonic/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, buildGoPackage, fetchFromGitHub
+, pkg-config, taglib, alsaLib
+
+# Disable on-the-fly transcoding,
+# removing the dependency on ffmpeg.
+# The server will (as of 0.11.0) gracefully fall back
+# to the original file, but if transcoding is configured
+# that takes a while. So best to disable all transcoding
+# in the configuration if you disable transcodingSupport.
+, transcodingSupport ? true, ffmpeg
+
+# udpater
+, writers, vgo2nix }:
+
+assert transcodingSupport -> ffmpeg != null;
+
+let
+  # update these, then run `updateScript` to update dependencies
+  version = "0.11.0";
+  rev = "056fb54a703ef5b5194ce112cbbdd8fb53dbb1ea";
```

Comment:
Why not use `"v${version}"`?

Reply:
Tags are unstable.

Reply:
Could you add a comment mentioning it then? Though it would be nicer to convince upstream to treat tags as immutable.

Reply:
Huh, the version tag is right above it. It’s a problem with git(hub), not with upstream.